scripts/project-sub-dir/README.md

```diff
@@ -0,0 +1,82 @@
+# Moving files to project subdirectories
+
+Per [DCP/2 MVP documentation](https://docs.google.com/document/d/1NsibP8g-NeLnksxlcBWQsSj5Zg_uimCDaAF1qB_qkjg/edit?ts=5ec6e28b&pli=1#heading=h.926y8csy88ms)
+> For new DCP/2 datasets wrangled by EBI/UCSC, each project will be treated as one source and will have its own staging area. All staging areas for such projects appear in the staging bucket under a common folder. There is no requirement that all new datasets are segregated into one staging area per project.
+
+> This will allow us to:
+>    * start the import even when not all projects are exported yet
+>    * import a small set and verify some things on the snapshot creation before doing everything
+>    * easily exclude projects from snapshots which are not ready for analysis yet
+>    * easily delete a project in the bucket
+
+Example paths for ingest areas:
+```
+(dev) gs://broad-dsp-monster-hca-dev-ebi-staging/dev/<project-uuid>
+(staging) gs://broad-dsp-monster-hca-dev-ebi-staging/staging/<project-uuid>
+(prod) gs://broad-dsp-monster-hca-prod-ebi-storage/prod/<project-uuid>
+
+```
+
+GitHub Issue: [ebi-ait/dcp-ingest-central#61](https://github.com/ebi-ait/dcp-ingest-central/issues/61)
+
+
+## Goal
+Group the existing files in the staging area to be in their own project directory
+
```

Comment:
Kind of missing a "pre-requisites" here, even if it just points out to the SOP on how to set up the gsutil configuration

Reply:
will add that! thanks!